examples/module-dependency-order.js

```diff
@@ -1,6 +1,11 @@
 // example - valid: true
 angular.module('myModule', ['ngAnimate', 'ngRoute', 'app', 'appFilters', 'ui.router']);
 
+// example - valid: true
+// with ES6 syntax:
+// import uiRouter from 'angular-ui-router';
```

Comment:
I tweaked it a bit but i think I'm missing something: https://github.com/jrencz/eslint-plugin-angular/commit/2fc22a31a2a901a6793855167884f2f83c8e8174

Let me know what do you think about this approach and if you have any comments. If you like it I'll create a PR for allowing defining testerConfig on per-example basis and after that one is merged this one can be (because I see no reason in making an example of what I did in this PR with variables, not imports)